src/System.Security.Cryptography.Primitives/src/System/Security/Cryptography/HashAlgorithm.cs

```diff
@@ -11,20 +11,20 @@ public abstract class HashAlgorithm : IDisposable, ICryptoTransform
     {
         private bool _disposed;
         protected int HashSizeValue;
-        protected internal byte[] HashValue;
+        protected internal byte[]? HashValue;
         protected int State = 0;
 
         protected HashAlgorithm() { }
 
-        public static HashAlgorithm Create() =>
+        public static HashAlgorithm? Create() =>
             CryptoConfigForwarder.CreateDefaultHashAlgorithm();
 
-        public static HashAlgorithm Create(string hashName) =>
-            (HashAlgorithm)CryptoConfigForwarder.CreateFromName(hashName);
+        public static HashAlgorithm? Create(string hashName) =>
```

Comment:
I don't particularly care for the zero-argument Create overloads that throw on core, but it requires a very mis-configured system for the zero-argument Create methods to return null.

So intent is non-null, the work has already been done to say maybe-null, and the methods are all PNSE in Core... so it doesn't really matter what we do here.

Reply:
Will update

Reply:
Oh, sorry, the `Create()` overloads (no parameters) can't return null, but the `Create(string)` ones can.

Reply:
oh i see, Create() methods loading with default name which would guaranteed to exist, so not null, but for Create(name) algorithm with user defined name might not exist